{"uuid": "df4a0d7d-abad-4b57-b43c-42b1d4f10f84", "children": ["08335760-364e-4caf-a45d-92180006514a", "ce1d29e5-21f5-4e95-99a2-8e6d0857bc23", "b5241752-37cb-47e2-9b06-795bd7ee7f39", "0d785cc3-b3ee-4fa3-bbb2-29ccd80fa744", "8f48e92e-3713-43bf-a197-7a1151cb8a7f", "d87234e1-819c-4da2-b54a-08f3e996daba", "bc172170-4c00-4c11-a4fc-06e5b1dc8006", "1c3e33cb-7b4a-4bff-8ab4-4823b2b281d7", "fe25a5a7-f3b8-494e-b437-48b103480a19"], "befores": [{"name": "driver_init", "status": "passed", "start": 1535632429208, "stop": 1535632432472}], "afters": [{"name": "driver_init::0", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"c:\\users\\dhimenes\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\allure_commons\\_allure.py\", line 169, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"c:\\users\\dhimenes\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 808, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\dhimenes\\PycharmProjects\\ssp_aqa_phyton_dhimenes\\conftest.py\", line 17, in driver_init\n    web_driver.close()\n  File \"c:\\users\\dhimenes\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 687, in close\n    self.execute(Command.CLOSE)\n  File \"c:\\users\\dhimenes\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 318, in execute\n    response = self.command_executor.execute(driver_command, params)\n  File \"c:\\users\\dhimenes\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 375, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"c:\\users\\dhimenes\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 397, in _request\n    resp = self._conn.request(method, url, body=body, headers=headers)\n  File \"c:\\users\\dhimenes\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\urllib3\\request.py\", line 68, in request\n    **urlopen_kw)\n  File \"c:\\users\\dhimenes\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\urllib3\\request.py\", line 89, in request_encode_url\n    return self.urlopen(method, url, **extra_kw)\n  File \"c:\\users\\dhimenes\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\urllib3\\poolmanager.py\", line 322, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"c:\\users\\dhimenes\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\urllib3\\connectionpool.py\", line 667, in urlopen\n    **response_kw)\n  File \"c:\\users\\dhimenes\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\urllib3\\connectionpool.py\", line 600, in urlopen\n    chunked=chunked)\n  File \"c:\\users\\dhimenes\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\urllib3\\connectionpool.py\", line 354, in _make_request\n    conn.request(method, url, **httplib_request_kw)\n  File \"c:\\users\\dhimenes\\appdata\\local\\programs\\python\\python37-32\\lib\\http\\client.py\", line 1229, in request\n    self._send_request(method, url, body, headers, encode_chunked)\n  File \"c:\\users\\dhimenes\\appdata\\local\\programs\\python\\python37-32\\lib\\http\\client.py\", line 1275, in _send_request\n    self.endheaders(body, encode_chunked=encode_chunked)\n  File \"c:\\users\\dhimenes\\appdata\\local\\programs\\python\\python37-32\\lib\\http\\client.py\", line 1224, in endheaders\n    self._send_output(message_body, encode_chunked=encode_chunked)\n  File \"c:\\users\\dhimenes\\appdata\\local\\programs\\python\\python37-32\\lib\\http\\client.py\", line 1016, in _send_output\n    self.send(msg)\n  File \"c:\\users\\dhimenes\\appdata\\local\\programs\\python\\python37-32\\lib\\http\\client.py\", line 956, in send\n    self.connect()\n  File \"c:\\users\\dhimenes\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\urllib3\\connection.py\", line 196, in connect\n    conn = self._new_conn()\n  File \"c:\\users\\dhimenes\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\urllib3\\connection.py\", line 171, in _new_conn\n    (self._dns_host, self.port), self.timeout, **extra_kw)\n  File \"c:\\users\\dhimenes\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\urllib3\\util\\connection.py\", line 69, in create_connection\n    sock.connect(sa)\n"}, "start": 1535632535628, "stop": 1535632537659}], "start": 1535632429208, "stop": 1535632537659}